process/block/metablock.go

```diff
@@ -1879,9 +1907,5 @@ func (mp *metaProcessor) getPendingMiniBlocks() []bootstrapStorage.PendingMiniBl
 		}
 	}
 
```

Comment:
why is this removed? 
are we sure that we can never return empty pendingMiniBocksInfo?
if these are afterwards serialized, there might be some issues